skills_DS/api/views.py

```
@@ -20,7 +26,74 @@ def post(self, request, format=None):
+
+class GetJobsView(APIView):
+	permission_classes = [IsAdminUser]
+
+	def post(self, request):
+		position = request.data['position']
+		location = request.data['location']
+		country = request.data['country']
+		remote = request.data['remote']
+		num = int(request.data['number'])
+		radius = int(request.data['radius'])
+		get_jobs(position, location, num, country, remote, radius)
+		return Response({'hey': 'it worked'}, status=status.HTTP_200_OK)
+
+
+class GetSkillsView(APIView):
+	permission_classes = [IsAdminUser]
+
+	def post(self, request):
+		position = request.data['position']
+		location = request.data['location']
+		distance = int(request.data['distance'])
+		try:
+			extract_skills(position, location, distance)
+			return Response({"success": "success"}, status=status.HTTP_200_OK)
+		except Exception as ex:
+			print(ex)
+			return Response({"error": str(ex)}, status=status.HTTP_400_BAD_REQUEST)
+
+
+class ListSkillsView(ListAPIView):
+	permission_classes = [IsAdminUser]
+	serializer_class = SkillSerializer
+
+	def get_queryset(self):
+		return Skill.objects.filter(verified=False)[:20]
+
+
+class UpdateSkillsView(APIView):
+	permission_classes = [IsAdminUser]
+	def post(self, request):
+		skills = request.data
+		for skill in skills:
+			job_title = JobTitle.objects.filter(name=skill['job_title'])
+			if len(job_title) == 0:
+				print("could not get job title")
+				continue
+			job_title = job_title[0]
+			query = Skill.objects.filter(name=skill['skill'], job_title=job_title)
+			if len(query) > 0:
+				query = query[0]
+				if skill['value'] == "good":
+					query.verified = True
+					query.save()
+				if skill['value'] == "invalid":
+					InvalidSkill.objects.get_or_create(job_title=job_title, name=skill['skill'], specific=False)
+					query.delete()
+				if skill['value'] == "invalid2":
+					InvalidSkill.objects.get_or_create(job_title=job_title, name=skill['skill'], specific=True)
+					query.delete()
+			else:
+				print("could not get skill")
```

This should return an error response.